pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -78,52 +81,85 @@ func (c *Cluster) makeDeployment(mgrConfig *mgrConfig, port int) *extensions.Dep
 	return d
 }
 
-func (c *Cluster) makeConfigInitContainer(mgrConfig *mgrConfig) v1.Container {
-	return v1.Container{
-		Name: opspec.ConfigInitContainerName,
+func (c *Cluster) makeCopyKeyringInitContainer(mgrConfig *mgrConfig) v1.Container {
+	// mgr does not obey `--keyring=/etc/ceph/keyring-store/keyring` flag, and always reports:
+	// "unable to find a keyring on /var/lib/ceph/mgr/ceph-a/keyring: (2) No such file or directory"
+	// Workaround: copy the keyring to the container data dir
+	container := v1.Container{
+		Name: "init-copy-keyring-to-data-dir",
```

Comment:
Would be good to remove the container if not needed as  @travisn wrote.

Reply:
Fix incoming. Want to run a local test too...